### PR TITLE
7.x code clean up show parent tn

### PR DESCRIPTION
**Pull request by nhart**

### CODE REVIEW

File: includes/blocks.inc [hunk 1]
```diff
@@ -108,3 +108,34 @@ function islandora_compound_object_jail_display_block() {
...
   return $block;
 }
+
+/**
+ * Constructs the default navigation block for a compound.
+ *
+ * @return array
+ *   Renderable array for the block.
+ */
+function islandora_compound_object_navigation_block() {
```
Comment (willtp87) [inline includes/blocks.inc:118]: Moving functions around like this is an API change and has to go through our usual process for such. There are options 1. Deprecation cycle 2. This is minimal risk so with commiters buy in we can forgo deprecation and just move it and put the move in the release notes. 3. Don't do it.

File: includes/manage.form.inc [hunk 2]
```diff
@@ -64,44 +64,43 @@ function islandora_compound_object_manage_form($form, &$form_state, $object) {
...
-    $parents = array();
-    foreach ($parent_part_of as $parent) {
-      // Shouldn't be too much of a hit but would be good to avoid the
-      // islandora_object_loads.
-      $pid = $parent['object']['value'];
-      $parent_object = islandora_object_load($pid);
-      $parents[$pid] = array('title' => $parent_object->label, 'pid' => $pid);
-    }
+    // Remove parents.
+    $parent_part_of = $object->relationships->get('info:fedora/fedora-system:def/relations-external#', $rels_predicate);
```
Comment (whikloj) [inline includes/manage.form.inc:80]: Should you be using the FEDORA_RELS_EXT_URI constant instead of a literal string?

Comment (nhart) [inline includes/manage.form.inc:80]: agreed

Comment (willtp87) [inline includes/manage.form.inc:80]: Pre-existing, another pull later if you want.

**Pull request merged**